README.md

```diff
@@ -321,21 +326,12 @@ response will be in the same order as the requests.
 
 ## Response parsing
 
-The response parser is the biggest difference between Eredis and other
-libraries like Erldis, redis-erl and redis_pool. The common approach
-is to either directly block or use active once to get the first part
-of the response, then repeatedly use `gen_tcp:recv/2` to get more data
-when needed. Profiling identified this as a bottleneck, in particular
-for `MGET` and `HMGET`.
-
-To be as fast as possible, Eredis takes a different approach. The
-socket is always set to active once, which will let us receive data
-fast without blocking the gen_server. The tradeoff is that we must
-parse partial responses, which makes the parser more complex.
-
-In order to make multibulk responses more efficient, the parser
-will parse all data available and continue where it left off when more
-data is available.
+The socket is set to `{active, N}` with N = 10 by default. This is configurable
+by including `{active, N}` in socket options or TLS options (if TLS is used). N
+must be an integer or `true`. Active `once` and `false` are not supported.
+
+The response parser is capable of parsing partial data and continuing when more
+data arrives on the socket.
```

Comment:
Maybe add the rationale saying something like

> customizing the `{active, N}` value allows fine performance tuning as a tradeoff between memory consumption and CPU consumption; 
> * a lower value will use more CPU (as each erlang port activation costs time)
> * a higher value - depending on how much data is sent by the peer - may increase memory usage for the client erlang process

Reply:
I think these are details of the OTP socket handling (like all the other TCP options) and can be studied in their docs. We do provide a sensible default for the protocol. We can add a link to e.g. https://www.erlang.org/doc/man/inet.html#setopts-2

Reply:
We already have a link to `gen_tcp` and `ssl` about the options. The gen_tcp page link to `inet:setopts/2`: "If the socket is in {active, N} mode (see [inet:setopts/2](https://www.erlang.org/doc/man/inet.html#setopts-2) for details)"